index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+
+      <div class="title-wrap">
+        <h1 id="title">Nikola Tesla</h1>
+      </div>
+
+      <div id="img-div">
+        <img id="image" src="images/nikola-tesla.jfif">
```

Comment:
You need to use the alt attribute in image tags because that is a good HTML semantic practice.  💯 
```suggestion
        <img id="image" src="images/nikola-tesla.jfif" alt="nikola-tesla">
```